native/acter/src/platform/native.rs

```diff
@@ -94,7 +95,7 @@ pub fn init_logging(
     if let Some(console_logger) = console_logger {
         builder = builder.chain(console_logger);
     } else {
-        builder = builder.chain(std::io::stdout());
+        builder = builder.chain(io::stdout());
```

Comment:
I can't quite work out the rules here ... `std::io::stdout()` is not nice, but `io::stdout()` is? Can you clarify the guide and its reasoning?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/837/commits/f505a9b2f23a2a43289e2b8ea73fd72128829fdc